test/plugins/sqlite.ts

```diff
@@ -37,7 +37,7 @@ describe("SQLite Message Storage (stateful tests)", function () {
 		// Delete database file from previous test run
 		await cleanup();
 
-		store = new MessageStorage("testUser");
+		store = new MessageStorage("testUser", undefined as any);
```

Comment:
such casts are very brittle... if you can avoid so please do and in this case you trivially could by providing a correct default value